sqeleton/databases/oracle.py

```diff
@@ -194,6 +194,19 @@ def _query_cursor(self, c, sql_code: str):
         except self._oracle.DatabaseError as e:
             raise QueryError(e)
 
+    def query_table_schema(self, path: DbPath) -> Dict[str, tuple]:
+        rows = self.query(self.select_table_schema(path), list)
+        if not rows:
+            raise RuntimeError(f"{self.name}: Table '{'.'.join(path)}' does not exist, or has no columns")
+
+        d = {r[0]: r for r in rows}
+        
+        # all oracle tables have a ROWID pseudocolumn
+        d['ROWID'] = ('ROWID', 'VARCHAR2', 6, None, None)
```

Comment:
A few extra concerns to discuss:

1. Adding this pseudo-column to the scanned schema leads to guaranteed false positives on schema comparison unless the other database is also Oracle. (But this pseudo-column will be ignored in value/row diffs in that case.)

2. Having 2 Oracle databases, leads to false positives anyway since in the new database, there will likely be different row ids even for the same rows — because it is a physical address of the row, as per their description. Except for the case of Fivetran, which explicitly takes and handles this column — but this seems to be overly specific.

Am I right in my assumptions? 

Can this column be made "optional" — i.e. recognized as existent but not used automatically?